extensions/elytron-security/deployment/src/test/java/io/quarkus/security/test/jdbc/CustomRoleDecoderTest.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.security.test.jdbc;
+
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+public class CustomRoleDecoderTest extends JdbcSecurityRealmTest {
```

Comment:
CustomRole is a cross-functionality of elytron-security, not sure it's useful to test it in with the jdbc auth mechanism as it should works for all mechanism.